src/algebra/group/pi.lean

```diff
@@ -147,6 +147,16 @@ end mul_hom
 
 section mul_hom
 
+/-- A family of monoid homomorphisms `f a : γ →ₙ* β a` defines a monoid homomorphism
+`pi.monoid_mul_hom f : γ →ₙ* Π a, β a` given by `pi.monoid_mul_hom f x b = f b x`. -/
+@[to_additive "A family of additive monoid homomorphisms `f a : γ →ₙ+ β a` defines a monoid
+homomorphism `pi.monoid_add_hom f : γ →ₙ+ Π a, β a` given by `pi.monoid_add_hom f x b
+= f b x`.", simps]
+def pi.monoid_mul_hom {γ : Type w} [Π i, has_mul (f i)] [has_mul γ]
```

Comment:
The name should be `pi.mul_hom`, or maybe `mul_hom.pi`, since nothing here is a monoid.

Also, I think you can add `simps`.

Reply:
I added a `monoid` version but I hope I didn't get confused by the different kind of maps. 

> Also, I think you can add simps.

Isn't the `simps` at the end of @[...,simps] doing that?

Note that I recycled the doctring from `pi.ring_hom` with, hopefully, the right modifications.

Reply:
Sorry, I didn't notice the final `simps`.